Source/Firefly.cpp

```diff
@@ -0,0 +1,29 @@
+#include "..\Include\Firefly.h"
+
+Firefly::Firefly(GLuint VAO, GLuint numVertices, glm::vec3 color)
+	: NonCollidableModel(VAO, numVertices, color) { 
+
+	glm::mat4 transformMatrix = glm::mat4(1.0f);
+	transformMatrix = glm::translate(transformMatrix, glm::vec3(0.0f, +5.0f, 0.0f));
+	transformMatrix = glm::scale(transformMatrix, glm::vec3(0.5f, 0.5f, 0.5f));
```

Comment:
I'd make the fireflies even smaller, right now they're chunky bois

Reply:
Done!